README.md

```diff
@@ -157,3 +164,27 @@ $ kubectl delete -f pvc.yaml
 persistentvolumeclaim "csi-devicepv" deleted
 ```
 
+Features
+---
+
```

Comment:
What would be in-scope other than meta-partition?

Reply:
meta partition is something that we use to identify the device.

@RealHarshThakur Did you mean creating volumes on a LUKS encrypted device => creating partitions on LUKS device

Reply:
@akhilerm No, I meant can we extend this repo to identify other block device types? Just thinking out loud : maybe it can be based on device mapper names so that it supports RAID, LUKS, etc.

Reply:
The initial thought is to have it get working for devices, may be we can have other types. But I think with little bit config change , currently it should support that.